boards/arm/nucleo_f091rc/doc/nucleof091rc.rst

```diff
@@ -142,27 +151,106 @@ This interface is supported by the openocd version included in the Zephyr SDK.
 Flashing an application to Nucleo F091RC
 ----------------------------------------
 
-Here is an example for the :ref:`blinky-sample` application.
+Here is an example for the :ref:`hello_world` application.
+
+First, run your favorite terminal program to listen for output.
+
+.. code-block:: console
+
+   $ minicom -D <tty_device> -b 115200
+
+Replace :code:`<tty_device>` with the port where the Nucleo F091RC board
+can be found. For example, under Linux, :code:`/dev/ttyACM0`.
+
+Then build and flash the application in the usual way.
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/basic/blinky
+   :zephyr-app: samples/hello_world
    :board: nucleo_f091rc
    :goals: build flash
 
-You will see the LED blinking every second.
-
 Debugging
 =========
 
-You can debug an application in the usual way.  Here is an example for the
-:ref:`hello_world` application.
+You can debug an application in the usual way. Here is an example for the
+:ref:`button-sample` application.
 
 .. zephyr-app-commands::
-   :zephyr-app: samples/hello_world
+   :zephyr-app: samples/basic/button
    :board: nucleo_f091rc
    :maybe-skip-config:
    :goals: debug
 
+Testing the Nucleo board
+************************
+
+Testing the button and the LED
+==============================
+
+There are 2 samples that allow you to test that the button and LED on
+the board are working properly with Zephyr:
+
+.. code-block:: console
+
+   samples/basic/blinky
+   samples/basic/button
+
+The button and LED definitions can be found in :file:`boards/arm/nucleo_f091rc/board.h`.
+
+You can build and flash the examples to make sure Zephyr is running correctly on
+your board. With the :ref:`blinky-sample` you will see the LED blinking every second.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: nucleo_f091rc
+   :goals: build flash
+
+Testing GPIO
+============
+
+To test the GPIO with loopback to itself a test harness must be set up
+(a jumper connecting CN10 pin 27 to CN10 pin 29).
+
++--------+--------+--------------------+
+| Signal | GPIO   | Connection on CN10 |
++========+========+====================+
+| IN     | PB4    | 27 <- 29           |
```

Comment:
I'm not sure what the arrow indicates here for a jumper?  Can you explain?

Reply:
Arrow shows signal from output to input.